scripts/needs_ci_run

```diff
@@ -128,6 +128,9 @@ for changed_file in changed_files:
         if fnmatch.fnmatch(changed_file, ignore_rule):
             ignored_files.append(changed_file)
 
+print(f"changed files: {changed_files}")
+print(f"ignored files: {ignored_files}")
```

Comment:
I actually didn't know this was still active.. I haven't ever seen it skip anything, I wanna see what it thinks are changed & ignored files